interop/PassStore.cpp

```diff
@@ -61,6 +61,16 @@ PassStore::PassStore() {
 			pathEntries.push_back(token);
 		}
 		for (const auto &dirName : pathEntries) {
+			fs::directory_iterator j;
+			// Handle the edge-case where a directory in
+			// $PATH doesn't exist.
+			try {
+				j = fs::directory_iterator(dirName);
+			} catch (fs::filesystem_error &e) {
+				std::cerr << "Path directory " << dirName
+					  << " doesn't exist, skipping" << std::endl;
+				continue;
+			}
```

Comment:
Is there an advantage to using this over fs::exists?

Reply:
My C++ is extremely rusty, I think it would probably be better to use fs::exists and elevate other kinds of filesystem errors (eg, $PATH entry not readable) to the user so they don't get misreported.